resources/assets/js/like.js

```diff
@@ -0,0 +1,58 @@
+var love = function() {
```

Comment:
sao lại viết kiểu này em?
```suggestion
function love() {
```